knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-resources.yaml

```diff
@@ -35,7 +21,7 @@ data:
       "annotations": {
         "list": []
       },
-      "description": "Knative Serving - Revision CPU and Memory Usage",
+      "description": "Knative Serving - Revision CPU, Memory and Network Usage",
```

Comment:
Same.

Reply:
It is too small and trivial. I noticed that when I did write the dashboard so I just added it.